activity-monitor.js

```diff
@@ -66,24 +67,38 @@
             keepAlive: null     /* setInterval handle for ping handler (options.frequency) */
         };
 
+        enable.timer = timer
```

Comment:
Mostly. When coming up with justification, I felt it could be handy for those of us who know what we are doing. Perhaps, it belongs as a naturally accessible variable? Again, thinking its use is for realtime debugging only, i left it semi hidden under the enable method.